tests/FakeItEasy.Tests/Configuration/RuleBuilderTests.cs

```diff
@@ -52,7 +52,7 @@ public void Returns_called_with_value_returns_parent_configuration()
 
             var result = returnConfig.Returns(10);
 
-            result.Should().Be(this.builder);
+            result.Should().BeSameAs(returnConfig);

```

Comment:
Are we still returning the "parent configuration"? I think that was `this.builder`.